react/Dialog/index.jsx

```diff
@@ -47,16 +48,17 @@ const Dialog = props => {
         },
     {
       bottomBackground: theme.palette.background[sidebar ? 'default' : 'paper'],
-      bottomTheme: 'dark',
+      bottomTheme: home ? 'light' : 'dark',
       bottomOverlay: 'transparent',
       topOverlay: 'transparent',
       topBackground:
         cozBar && getComputedStyle(cozBar).getPropertyValue('background-color'),
       topTheme:
-        cozBar && cozBar.classList.contains('coz-theme-primary')
+        home || (cozBar && cozBar.classList.contains('coz-theme-primary'))
```

Comment:
this feature deserves an explanation + a test

Reply:
nouvelle API plus claire utilisée ✅